openlibrary/templates/books/edit/edition.html

```diff
@@ -451,7 +451,7 @@
                                 <input type="hidden" name="{{prefix}}identifiers--{{index}}--name" value="{{name}}"/>
                                 <input type="hidden" name="{{prefix}}identifiers--{{index}}--value" value="{{value}}" class="{{name}}"/>
                             </td>
-                            <td><a href="javascript:;" class="repeat-remove red plain" title="Remove this identifier">[x]</a></td>
+                            <td><a href="javascript:;" class="repeat-remove red plain" title="$_('Remove this identifier')">[x]</a></td>
```

Comment:
Ah I don't think this can be i18n'd right now; it looks like this html is rendered via the JS in a peculiar way.

```suggestion
                            <td><a href="javascript:;" class="repeat-remove red plain" title="Remove this identifier">[x]</a></td>
```